base/_typesizes.scss

```diff
@@ -63,6 +63,18 @@ $type-sizes: (
 // ! ===== Mixins to support pulling out type sizes at breakpoints =====
 
 //
+// Deep getter to go into the $type-sizes array
+// and return with a specific element at a particular breakpoint
+// @jhogue
+//
+// @usage: .bravo { font-size: type-size(h2,large); }
```

Comment:
Hang on, thats not a file that I changed in this branch. Let me investigate the source and get back to you.